app/assets/javascripts/angular/directives/graph_chart.js

```diff
@@ -389,6 +384,15 @@ angular.module("Prometheus.directives").directive('graphChart', ["$location", "W
         return "<table class=\"labels_table\">" + labelRows.join("") + "</table>";
       }
 
+      function storeLegendState() {
+        if (!rsGraph) {
+          return;
+        }
+        rsGraph.series.forEach(function(s) {
```

Comment:
Is this repeating that pattern from the previous PR where we store all the state instead of just the disabled ones in the map? Please don't do that, as there can be gazillions of series in a single dashboard, and we don't want to store them all in the graph JSON, only the disabled ones, if any.